### PR TITLE
Allow opening up the feedback tab from URL query params

### PR DESCRIPTION
**Pull request by ClementPasteau**
useful for emails when there is a new feedback on the game without wasting time finding the game dashboard & feedbacks tab.

https://user-images.githubusercontent.com/4895034/203592420-bf692d14-36eb-4965-b8cc-70d0da44cd9e.mov

https://user-images.githubusercontent.com/4895034/203592449-2c1acf54-e952-4df2-a394-a44648f5d722.mov

### CODE REVIEW

File: newIDE/app/src/BrowserApp.js [hunk 1]
```diff
@@ -70,6 +70,8 @@ export const create = (authentication: Authentication) => {
                 <BrowserS3PreviewLauncher {...props} ref={ref} />
               )}
               initialDialog={appArguments['initial-dialog']}
+              initialGameId={appArguments['game-id']}
+              initialGameTab={appArguments['game-tab']}
```
Comment (AlexandreSi) [inline newIDE/app/src/BrowserApp.js:74]: I feel like we should be more accurate: `game-dashboard-tab`?

File: newIDE/app/src/MainFrame/index.js [hunk 2]
```diff
@@ -521,14 +534,29 @@ const MainFrame = (props: Props) => {
 
   React.useEffect(
     () => {
-      if (initialDialog === 'subscription') {
-        openSubscriptionDialog(true);
-      }
-      if (initialDialog === 'onboarding') {
-        openOnboardingDialog(true);
+      console.log(initialDialog, initialGameId);
```
Comment (AlexandreSi) [inline newIDE/app/src/MainFrame/index.js:537]: To remove

File: newIDE/app/src/MainFrame/index.js [hunk 3]
```diff
@@ -521,14 +534,29 @@ const MainFrame = (props: Props) => {
 
   React.useEffect(
```
Comment (AlexandreSi) [inline newIDE/app/src/MainFrame/index.js:535]: Can I suggest that this effect goes in its own custom hook to lighten a bit MainFrame?

Comment (AlexandreSi) [reply newIDE/app/src/MainFrame/index.js:535]: And its own state variables

File: newIDE/app/src/BrowserApp.js (hunk 1 above)
Comment (4ian) [reply newIDE/app/src/BrowserApp.js:74]: Agree

File: newIDE/app/src/MainFrame/index.js (hunk 3 above)
Comment (4ian) [reply newIDE/app/src/MainFrame/index.js:535]: Good idea, some "useInitialState" or something like this!

**Review by AlexandreSi — APPROVED** [review]
LGTM!

**Pull request merged**